builtin/bisect--helper.c

```diff
@@ -339,6 +340,55 @@ static int bisect_next_check(const struct bisect_terms *terms,
 	return res;
 }
 
+static int get_terms(struct bisect_terms *terms)
+{
+	struct strbuf str = STRBUF_INIT;
+	FILE *fp = NULL;
+	int res = 0;
+
+	fp = fopen(git_path_bisect_terms(), "r");
+	if (!fp) {
+		res = -1;
+		goto finish;
```

Comment:
We could just return -1 here instead of using a goto. `str` has not been used yet, so we don't need to release it and we know that `fp`is NULL so we don't need to do anything with it.

Reply:
On the other hand, we would save ourselves some thinking if we introduce things in the future that *will* need to be released. In the interest of keeping this code easy to extend, I actually rather would keep the `goto finish`.

Reply:
Ok let's keep it as is.